bciers/libs/types/.eslintrc.json

```diff
@@ -1,8 +1,8 @@
 {
-  "extends": ["plugin:@nx/react", "../../../.eslintrc.json"],
+  "extends": ["plugin:@nx/react", "../../.eslintrc.json"],
   "ignorePatterns": ["!**/*"],
   "parserOptions": {
-    "project": ["libs/shared/actions/tsconfig(.*)?.json"],
+    "project": ["libs//types/tsconfig(.*)?.json"],
```

Comment:
I think is just a github change display thing, but is it right that this was previously actions and is now types?

Reply:
Hmm yeah I think this is some weird issue with how things were renamed. Looking at both `types` and `actions` eslint in this branch they are pointed at the correct `tsconfig` for each project.